Dockerfile.rhel

```diff
@@ -31,9 +34,19 @@ EXPOSE 80
 RUN chmod a+rwX            /etc/httpd/conf
 RUN chmod a+rwX            /run/httpd
 
-RUN mkdir /var/www/html/v3
-COPY --from=builder /build/ /var/www/html/
+COPY .htaccess README.md /var/www/html/
+COPY --from=builder /build/v3 /var/www/html/v3
 
 STOPSIGNAL SIGWINCH
 
 ENTRYPOINT ["/usr/sbin/httpd", "-D", "FOREGROUND"]
+
+# Offline build: cache .theia and .vsix files in registry itself and update metas
+FROM builder AS offline-builder
+RUN ./cache_artifacts.sh v3
+
+# Offline registry: copy updated meta.yamls and cached extensions
+FROM registry AS offline-registry
+COPY --from=offline-builder /build/v3 /var/www/html/v3
+USER 0
+RUN chmod -R g+rwX /var/www/html/v3/
```

Comment:
can you add a `WORKDIR /var/www/html/` to the end of the Dockerfile* files so that if you connect to the image you're already in the folder where all the magic happens? 

(rather than being in `/opt/app-root/src` where nothing magical happens)